### ISSUE TITLE
Batch Download

### ISSUE THREAD

**Issue by miladmozafari** [microsoft/vscode-remote-release#4685]
When I select multiple files to download from the remote machine to my local machine, it asks "where to save" for each of the files. Moreover, it does not remember the saving address of the last file and for each of them, I have to navigate to the target folder from scratch. Is it possible to add a feature for easier "batch download"?

**Comment by isidorn** [comment]
Yeah ideally you should just choose once, and not for every file.
We can look into this in the future.

**Comment by RMeli** [comment]
+1

This would be a great feature indeed!
